src/test-helper.js

```diff
@@ -0,0 +1,40 @@
+import TestUtils from 'react-addons-test-utils';
+import equals from 'lodash/isEqual';
+import ShallowTestUtils from 'react-shallow-testutils';
+
+const renderer = TestUtils.createRenderer();
```

Comment:
If we run tests in parallel will having a single renderer that is mutated affect our tests?